plugin.audio.ilovemusic/addon.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+import xbmcplugin,xbmcaddon,xbmcgui,os,sys
+
+addon = xbmcaddon.Addon()
+addon_path = addon.getAddonInfo('path').decode('utf-8')
```

Comment:
It would be wise to already prepare for Kodi Matrix and only decode if you received bytes:
```
addon_path = addon.getAddonInfo('path')
if isinstance(addon_path, bytes):
    addon_path = addon_path.decode('utf-8')
```